tools/prepare-source.sh

```diff
@@ -2,26 +2,25 @@
 
 set -eux
 
-# fetch latest thingy
+# Curl latest composer v2 spec
+curl https://raw.githubusercontent.com/osbuild/osbuild-composer/main/internal/cloudapi/v2/openapi.v2.yml \
+     -o internal/composer/openapi.v2.yml
 
-curl https://raw.githubusercontent.com/osbuild/osbuild-composer/main/internal/cloudapi/v2/openapi.v2.yml -o internal/composer/openapi.v2.yml
-
-# go stuff
-
-GO_VERSION=1.14.14
+GO_VERSION=1.15.13
```

Comment:
ubi has go 1.16, so we might even go with this version:

```
$ podman run --rm -it ubi8/go-toolset go version
go version go1.16.7 linux/amd64
```

Reply:
ah forgot to pull beforehand :facepalm: fixing